### PR TITLE
DOC: cleanup build warnings

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

File: doc/source/dsintro.rst [hunk 1]
```diff
@@ -935,134 +935,20 @@ method:
...
-    # print a slice of our 5D
-    p5d.ix['C1',:,:,0:3,:]
-
-    # transpose it
-    p5d.transpose(1,2,3,4,0)
-
-    # look at the shape & dim
-    p5d.shape
-    p5d.ndim
+See the `docs of a previous version <http://pandas.pydata.org/pandas-docs/version/0.18.1/dsintro.html#panel4d-experimental>`__
```
Comment (jreback) [inline doc/source/dsintro.rst]: +1

File: doc/source/dsintro.rst [hunk 2]
```diff
@@ -935,134 +935,20 @@ method:
...
-.. ipython:: python
-
-   p4d.transpose(3, 2, 1, 0)
-
 .. _dsintro.panelnd:
+.. _dsintro.panel4d:
 
-PanelND (Experimental)
-----------------------
+Panel4D and PanelND (Experimental)
```
Comment (jreback) [inline doc/source/dsintro.rst]: maybe Experimental -> Deprecated?

File: doc/source/sparse.rst [hunk 3]
```diff
@@ -97,6 +97,7 @@ of SparseArrays. To create one, simply call the ``SparseList`` constructor with
 a ``fill_value`` (defaulting to ``NaN``):
 
 .. ipython:: python
+   :okwarning:
 
    spl = pd.SparseList()
```
Comment (jreback) [inline doc/source/sparse.rst]: I would put a pointer to previous docs as this is now deprecated

**Comment by jreback** [comment]
minor comments, lgtm.

File: doc/source/dsintro.rst (hunk 2 above)
Comment (jorisvandenbossche) [inline doc/source/dsintro.rst]: ah, yes :-)

File: doc/source/sparse.rst (hunk 3 above)
Comment (jorisvandenbossche) [inline doc/source/sparse.rst]: done

**Pull request merged**